src/components/ListItemIcons/ListItemIconBase/__test__/ListItemIconBase.test.tsx

```diff
@@ -28,7 +29,7 @@ describe("<ListItemIconBase />", () => {
         props = {
             children: "Test",
             color: "primary",
-            bgColor: "primary",
+            bgcolor: "primary",

```

Comment:
Why?

Reply:
Does box use bgcolor?

Reply:
yes, box using "bgcolor" instead of "bgColor" so I decided to change so that if anyone use our component will be not confuse in the typo.